packages/utils/src/css.ts

```diff
@@ -0,0 +1,38 @@
+//
+// DOM - CSS
+//
+
+import { arrayForEach, addOrRemoveItem } from './array';
+
+// For details on the pattern for changing node classes
+// see: https://github.com/knockout/knockout/issues/1597
+const cssClassNameRegex = /\S+/g;
+
+function toggleDomNodeCssClass(node: HTMLElement, classNames: string, shouldHaveClass?: boolean) {
+  let addOrRemoveFn: (className: string) => void;
+  if (!classNames) { return; }
+  if (typeof node.classList === 'object') {
+    addOrRemoveFn = node.classList[shouldHaveClass ? 'add' : 'remove'];
+    arrayForEach(classNames.match(cssClassNameRegex)!, (className: string) => {
+      addOrRemoveFn.call(node.classList, className);
+    });
+  } else if (typeof (node.className as any).baseVal === 'string') {
+        // SVG tag .classNames is an SVGAnimatedString instance
+    toggleObjectClassPropertyString(node.className, 'baseVal', classNames, shouldHaveClass);
```

Comment:
IE < 10 isn't a target anymore right? Perhaps redirect directly to `classlist` instead of attempting to feature-detect and workaround?

Reply:
Yeah, there's a ton of pruning we could do for non-old-IE, including this.